index.js

```diff
@@ -0,0 +1,38 @@
+function mySort(param){
+    for (let i = 0 ; i < param.length ; i++){
+        for (let j = 0 ; j < param.length - i - 1; j++){
+            if (param[j] > param[j+1]){
+                let temp = param[j];
+                param[j] = param[j+1];
+                param[j+1] = temp;
+            }
+        }
+    }
+}
+
+function generateBoard(param){
+    let result = [];
+    let start = (param * param);
+
+    for (let i = 0 ; i < param ; i++){
+        let container = [];
+        for (let j = 0 ; j < param ; j++){
+            container.push(start);
+            start -= 1;
+        }
+        if (param % 2 === 1 && i % 2 === 0){
```

Comment:
line 23 dan 26 bisa digabung jadi 1 kondisi saja dengan operator ||
```suggestion
        if ((param % 2 === 1 && i % 2 === 0) || (param % 2 === 0 && i % 2 === 1)){
```